main/exercice/fill_blanks.class.php

```diff
@@ -190,11 +401,503 @@ function processAnswersCreation($form)
      */
     function return_header($feedback_type = null, $counter = null, $score = null)
     {
-	    $header = parent::return_header($feedback_type, $counter, $score);
-	    $header .= '<table class="'.$this->question_table_class .'">
-			<tr>
+        $header = parent::return_header($feedback_type, $counter, $score);
+        $header .= '<table class="'.$this->question_table_class .'">
+            <tr>
                 <th>'.get_lang("Answer").'</th>
-			</tr>';
+            </tr>';
         return $header;
-	}
+    }
+
+   /**
+     * @param $separatorStartRegexp
```

Comment:
Make sure you properly define the goal of your function as part of the PHPDoc block